Makefile

```diff
@@ -167,7 +167,7 @@ proto-mocks: $(PROTO_OUT)
 	@printf $(COLOR) "Generate proto mocks..."
```

Comment:
Btw I considered trying to move these into go:generate too. But then I really had to move all of the proto stuff, and that's much more complicated